### PR TITLE
Ui/dropdown

### LINKED ISSUE #964: [Enhancement] - Dropdown width and placement
*Prioritized*
- We would like this enhancement to have high priority as we need it for a feature that we hoped to release in this sprint and enhance upon in the next sprint

*UX Refined*
- https://zpl.io/aB8ddDK

*Tech Refined*
- We'll do a quick bugfix for the placement issue: When the dropdown is aligned to the right side of the screen and the left offset+the width of the dropdown options is wider than the width of the screen, the dropdown options should be right aligned.
- The min-width will be explored in different enhancement issue: #940

**Is your enhancement request related to a problem? Please describe.**

1. The dropdown i as default placed so that the left side aligns with the left side of the button. This makes the dropdown go off the right of the screen when the dropdown is placed at the right side of the screen.
2. The min width of the dropdown is quite wide leaving a lot of empty space in some of our dropdowns. 

**Describe the solution you'd like**
1. The possibility to set right or left alignment. Default left (as is) and right should make the right side of the dropdown align with the right side of the button.
2. A smaller min width or width adjusted to content. Not as important as number 1.

**Describe alternatives you've considered**

**Additional context**
What we would like
Screenshot 2020-03-23 at 13 40 31

Current behaviour
Screenshot 2020-03-23 at 13 42 16

### PR DESCRIPTION
**Pull request by jakobe**
This PR closes #964 

## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] Tests for the changes have been added (for bug fixes / features)
- [X] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [X] Bugfix
- [X] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] cookbook application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

If the dropdown is aligned to the right side of the screen the options will be cut off to the right when opened:
image

Issue Number: #964 

## What is the new behavior?
The dropdown options should be right aligned when otherwise cut off - same applies when aligned to the bottom:
image

## Does this PR introduce a breaking change?

- [ ] Yes
- [X] No

### CODE REVIEW

File: libs/designsystem/src/lib/components/dropdown/dropdown.component.ts [hunk 1]
```diff
@@ -158,6 +181,59 @@ export class DropdownComponent implements AfterContentChecked, OnDestroy, Contro
...
+      const callback: IntersectionObserverCallback = (entries) => {
+        // Only apply alignment when opening:
+        if (!this._isOpening) {
+          return;
+        }
+        const entry = entries[0];
+        const isVisible = entry.boundingClientRect.width > 0;
+        if (isVisible && entry.intersectionRatio < 1) {
+          // entry not fully showing:
+          // console.log('entry not fully showing:', entry);
```
Comment (SilverGull) [inline libs/designsystem/src/lib/components/dropdown/dropdown.component.ts]: Are all the console logs kept for future use? Consider deleting them if not :-)

File: libs/designsystem/src/lib/scss/themes/_colors.scss [hunk 2]
```diff
@@ -28,6 +28,8 @@ $text-colors: (
   'danger': #ee0d0d,
 );
 
+$focus-ring-color: rgb(77, 144, 254);
```
Comment (SilverGull) [inline libs/designsystem/src/lib/scss/themes/_colors.scss]: not a kirby color? I expect you have your reasons so I have approved anyway :-)

File: libs/designsystem/src/lib/components/dropdown/dropdown.component.ts (hunk 1 above)
Comment (jakobe) [reply libs/designsystem/src/lib/components/dropdown/dropdown.component.ts]: :fire: Removed :)

File: libs/designsystem/src/lib/scss/themes/_colors.scss (hunk 2 above)
Comment (jakobe) [reply libs/designsystem/src/lib/scss/themes/_colors.scss]: It IS a Kirby color added to the `_colors.scss` but it's utility color, not a brand color :)

**Pull request merged**